tailwind.config.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  theme: {
+    extend: {
```

Comment:
In this spike, I used `extend` to extend tailwind's default configuration, which already includes like a billion colors. We'd likely want to _not_ extend but instead define the entire theme to prevent style bloat.